account_invoice_refund_link/models/account_invoice.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2004-2011 Pexego Sistemas Informáticos. (http://pexego.es)
```

Comment:
File must be renamed to `account_move.py`

Reply:
done